packages/rke2-canal/charts/values.yaml

```diff
@@ -71,6 +71,10 @@ calico:
   typhaServiceName: none
   # Kubelet flex-volume-plugin-dir
   flexVolumePluginDir: /var/lib/kubelet/volumeplugins
+  # calico inbound failsafe ports. Empty string means defaults. Use 'none' to disable failsafe if you have your own rules.
+  FailsafeInboundHostPorts: ""
```

Comment:
I believe this should probably be prefixed with a `felix` and formatted in `camelCase` to keep with chart convention.

Reply:
Fixed